docs/source/manual.rst

```diff
@@ -0,0 +1,33 @@
+User Manual for ``autojudge``
+=============================
+
+Creating your account / Logging in
+----------------------------------
+
+On the home page, click ``LOG IN`` (see top right corner).
+Login via your IITH Google account.
+
+Basic Ideas for User
+--------------------
+
+The judge uses a system of ``Contest`` s.
+A ``Contest`` consists of a set of ``Problem`` s.
+
+A user can be a ``Poster``, a ``Participant`` or neither for a ``Contest``.
+For each ``Contest`` a user will hold one and only one position.
+
+The user who creats a ``Contest`` becomes the ``Poster`` for the ``Contest``.
+The ``Poster`` can add more ``Poster`` s, who would help in creating the problems.
+These ``Poster`` s together write a set of ``Problem`` s for the ``Contest``.
+
+The ``Poster`` can leave the ``Contest`` open for all users to particpate, in which case it is called a ``Public Contest``.
+Or the ``Poster`` may specify a set of ``Participant`` s. In this case only these users can particpate in the ``Contest``. Such ``Contest`` s are also called ``Private Contest`` s.
+Note that no ``Poster`` can ever participate in a contest.
+
+If the ``Contest`` is ``Public`` every user is either a ``Poster`` or a ``Participant``.
+If the ``Contest`` is ``Private``, a user can be either a ``Poster`` or a ``Participant`` or neither. All the users who are not specified as ``Participant`` by the ``Poster`` s belong to the category of neither. Such users cannot participate in that ``Contest``.
```

Comment:
Could be rephrased better: users who are neither participants nor posters in a private contest will not be allowed to participate in a contest.